modules/indexing/src/test/java/org/apache/ignite/cache/query/IndexQueryTestSuite.java

```diff
@@ -46,7 +46,8 @@
     ThinClientIndexQueryTest.class,
```

Comment:
Let's test the functionality with thin client too.

Reply:
while adding test discovered that thin client didn't pass the limit parameter, resolved that